sdcm/nemesis.py

```diff
@@ -4946,3 +4972,32 @@ def __init__(self, *args, **kwargs):
         super(SisyphusMonkey, self).__init__(*args, **kwargs)  # pylint: disable=bad-super-call
         self.build_list_of_disruptions_to_execute(nemesis_selector=['free_tier_set'])
         self.shuffle_list_of_disruptions()
+
+
+class SlaIncreaseSharesDuringLoad(Nemesis):
+    disruptive = False
+    sla = True
+
+    def disrupt(self):
+        self.disrupt_sla_increase_shares_during_load()
+
+
+class SlaDecreaseSharesDuringLoad(Nemesis):
+    disruptive = False
+    sla = True
+
+    def disrupt(self):
+        self.disrupt_sla_decrease_shares_during_load()
+
+
+class SlaNemeses(Nemesis):
```

Comment:
Why is this needed?
Why isn't it just a SisyphusMonkey with nemesis_selector = ['sla']?

Reply:
changed

Reply:
Actually we want to ral parallel nemesis: sla nemeses and in parallel different disruptive nemeses.
`nemesis_selector` feature does not support parallel nemeses, so I added this class back.
Maybe we need to support it

Reply:
@juliayakovlev lets raise an issue/task for parallel nemesis to better support `nemesis_selector`

Reply:
> @juliayakovlev lets raise an issue/task for parallel nemesis to better support `nemesis_selector` or any other nemesis configuration

@fruch 
https://github.com/scylladb/qa-tasks/issues/967